clang-tools-extra/clangd/XRefs.cpp

```diff
@@ -2346,11 +2346,11 @@ outgoingCalls(const CallHierarchyItem &Item, const SymbolIndex *Index) {
     // Filter references to only keep function calls
     using SK = index::SymbolKind;
     auto Kind = Callee.SymInfo.Kind;
-    if (Kind != SK::Function && Kind != SK::InstanceMethod &&
-        Kind != SK::ClassMethod && Kind != SK::StaticMethod &&
-        Kind != SK::Constructor && Kind != SK::Destructor &&
-        Kind != SK::ConversionFunction)
-      return;
+    bool NotCall = (Kind != SK::Function && Kind != SK::InstanceMethod &&
+                    Kind != SK::ClassMethod && Kind != SK::StaticMethod &&
+                    Kind != SK::Constructor && Kind != SK::Destructor &&
+                    Kind != SK::ConversionFunction);
+    assert(!NotCall);
```

Comment:
Can we get rid of the double negation? It just seems to obfuscate the logic for no particular reason.